packages/language-server/src/ui5-model.ts

```diff
@@ -11,29 +13,81 @@ import { Fetcher } from "../api";
 
 const DEFAULT_UI5_VERSION = "1.71.14";
 
-export async function getSemanticModel(): Promise<UI5SemanticModel> {
-  return getSemanticModelWithFetcher(fetch);
+export async function getSemanticModel(
+  modelCachePath: string | undefined
+): Promise<UI5SemanticModel> {
+  return getSemanticModelWithFetcher(fetch, modelCachePath);
 }
 
 // This function is exported for testing purposes (using a mock fetcher)
 export async function getSemanticModelWithFetcher(
-  fetcher: Fetcher
+  fetcher: Fetcher,
+  modelCachePath: string | undefined
 ): Promise<UI5SemanticModel> {
   const version = DEFAULT_UI5_VERSION;
   const jsonMap: Record<string, Json> = {};
   const baseUrl = `https://sapui5.hana.ondemand.com/${version}/test-resources/`;
   const suffix = "/designtime/api.json";
   const libs = getLibs();
+  let cacheFolder: string | undefined;
+
+  // Note: all cache handling (reading, writing etc) is optional from the user perspective but
+  // impacts performance, therefore if any errors occur when handling the cache we ignore them but output
+  // a warning to the user
```

Comment:
- Where do these console.logs go? (The extension host terminal?)
- Could we get these logs to the output channel?
- I think I saw some APIs to log via the connection somewhere...

Reply:
They go to the output channel in vscode with the defined extension name (`UI5 Language Assitant`). Same as `connection.console`.